### PR TITLE
Closes #512

### LINKED ISSUE #512: Edit regex pattern for 'invite_cmd' in 'labhub.py'
The `invite_cmd` method in `labhub.py` has a particular regex pattern which should match *'invite @username to team'*, *'invite @username'* or *'invite me'*. However, the regex pattern might also match to *'invite @userto developers'* while match.group(1) will return *'user'* (instead of *'userto'*) and should be changed to remove a possible corner-case error.

### PR DESCRIPTION
**Pull request by random-access7**
labhub: Change regex pattern for 'invite_cmd'

The original file had regex for `\s*` which matches 0 or more whitespaces which led to matches such as *'invite @userto developers'* since it matched 0 whitespaces between *'@user'* and *'to'*. With `\s+` it matches 1 or more whitespace and hence *'invite @userto developers'* will not be matched. Added a test to verify this as well.

Fixes https://github.com/coala/corobo/issues/512
# Reviewers Checklist
- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

**Comment by nemaniarjun** [comment]
Hey! Thanks for this contribution, a few minor things though, can we have tests that ensure this is working as expected?
Also commit messages should follow the guideline set in coala.io/commit

**Comment by random-access7** [comment]
@nemaniarjun , introduced the requisite test in `labuhub_tests.py` and squashed everything into a single commit as well.

File: tests/labhub_test.py [hunk 1]
```diff
@@ -55,9 +55,11 @@ def test_invite_cmd(self):
                                    'select from one of the')
 
         self.mock_team.is_member.return_value = False
-
```
Comment (Vamshi99) [inline tests/labhub_test.py]: Why did you remove this line? It is an irrelevant change.

File: tests/labhub_test.py [hunk 2]
```diff
@@ -55,9 +55,11 @@ def test_invite_cmd(self):
...
         testbot.assertCommand('!invite meet to developers',
                                    ':poop:')
+        self.mock_team.is_member.return_value = False
```
Comment (Vamshi99) [inline tests/labhub_test.py]: No need of this statement, is member return value is already set to False

**Comment by Vamshi99** [comment]
unack 662884d

File: tests/labhub_test.py (hunk 1 above)
Comment (random-access7) [reply tests/labhub_test.py]: Apologies for the mistake. Correcting at once.

**Comment by Vamshi99** [comment]
ack b1eaf58

**Comment by random-access7** [comment]
Changing by resolving conflicts with master branch.

File: tests/labhub_test.py [hunk 3]
```diff
@@ -58,7 +58,8 @@ def test_invite_cmd(self):
 
         testbot.assertCommand('!invite meet to developers',
                                    ':poop:')
-
```
Comment (Vamshi99) [inline tests/labhub_test.py]: Irrelevant change.

**Comment by random-access7** [comment]
Changes made @Vamshi99

**Comment by nemaniarjun** [comment]
LGTM!

**Comment by Makman2** [comment]
Code looks good, but please use now `Fixes` in your commit message instead of `Closes`, as the issue is tagged with `bug` ;)

**Comment by Makman2** [comment]
unack f19f9c06f29adaabb04cc2b095e7d897b8aa5113

**Comment by Makman2** [comment]
ack ec452298704d36a98b1c64ca449bc27b48cd56f6

**Comment by Makman2** [comment]
@gitmate-bot ff
